aidants_connect_web/tests/test_models.py

```diff
@@ -124,6 +125,55 @@ def test_normalize_birthplace(self):
         self.assertEqual(usager.birthplace, "12345")
 
 
+@tag("models")
+class MandatModelTests(TestCase):
+    def setUp(self):
+        self.organisation_1 = OrganisationFactory()
+        self.aidant_1 = AidantFactory(username="aidants1@organisation1.com")
+
+        self.usager_1 = UsagerFactory()
+        self.mandat_1 = Mandat.objects.create(
+            organisation=self.organisation_1,
+            usager=self.usager_1,
+            creation_date=timezone.now(),
+            duree_keyword="SHORT",
+            expiration_date=timezone.now() + timedelta(days=1),
+        )
+        self.autorisation_1 = AutorisationFactory(
+            aidant=self.aidant_1, usager=self.usager_1, mandat=self.mandat_1
+        )
+        self.usager_2 = UsagerFactory()
+        self.mandat_2 = Mandat.objects.create(
+            organisation=self.organisation_1,
+            usager=self.usager_2,
+            creation_date=timezone.now(),
+            duree_keyword="SHORT",
+            expiration_date=timezone.now() + timedelta(days=1),
+        )
+
+        self.autorisation_2 = AutorisationFactory(
+            aidant=self.aidant_1,
+            usager=self.usager_1,
+            mandat=self.mandat_2,
+            demarche="argent",
+        )
+        self.autorisation_3 = AutorisationFactory(
+            aidant=self.aidant_1,
+            usager=self.usager_1,
+            mandat=self.mandat_2,
+            demarche="transport",
+        )
+
+    def test_saving_and_retrieving_mandats(self):
```

Comment:
Je ne suis pas sûr de voir l'utilité de ces 3 tests, dans le sens où on n'a pas implémenté de comportement spécifique sur le `save` du mandat. J'ai l'impression que c'est essentiellement "tester que Django marche", donc quelque part redondant avec les tests du framework lui-même, mais peut-être que je ne vois pas quelque chose ? :)

Reply:
Pour moi, le test montre comment on créé un mandat  et les fonctionnalité pratique (comme pouvoir en récupérer toutes les autorisations. Peut-être que les deux et trois sont redondants en effet.

Reply:
D'accord, pour reboucler sur notre discussion d'hier, c'est davantage du "test as documentation" que du "test to prove it works", en quelque sorte ? :)